rules/windows/collection_email_powershell_exchange_mailbox.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/12/15"
+maturity = "production"
+updated_date = "2020/12/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of PowerShell Exchange New-MailBoxExportRequest Command to export content of a primary mailbox or archive
+to a .pst file. Adversaries may target user email to collect sensitive information.
+"""
+false_positives = ["Legitimate exchange system administration activity."]
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Exporting MailBox with PowerShell"
```

Comment:
We might want to call this something like "Exporting Exchange MailBox via PowerShell" to differentiate other kinds of mailboxes.